pkg/sheets/sheets.go

```diff
@@ -1,66 +1,154 @@
 package sheets
 
 import (
+	"bytes"
+	"context"
+	"crypto/md5"
```

Comment:
```suggestion
	"crypto/sha256"
```

Reply:
md5 hash is good enough for a pdf file name.